helm-chart/amalthea/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.1-rc1
```

Comment:
I would either put this back the way it was or change it to `0.1.0-rc11`. I will submit a PR to tag a proper release.